libs/ledger-live-common/src/families/bitcoin/wallet-btc/storage/index.ts

```diff
@@ -78,13 +78,10 @@ class BitcoinLikeStorage implements IStorage {
       if (this.txs[this.primaryIndex[index]]) {
         return;
       }
-      if (
-        typeof this.accountIndex[`${tx.account}-${tx.index}`] === "undefined"
-      ) {
-        this.accountIndex[`${tx.account}-${tx.index}`] = [];
-      }
       const idx = this.txs.push(tx) - 1;
       this.primaryIndex[index] = idx;
+      this.accountIndex[`${tx.account}-${tx.index}`] =
+        this.accountIndex[`${tx.account}-${tx.index}`] || [];
```

Comment:
`??` is safer than `||` in this kind of case, but probably not very important here.

Reply:
It is the same here. but as in the file, we use all "||" to handle this kind of logic. I usee "||" here too